src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests_Attributes_FullyQualifiedName.cs

```diff
@@ -1350,6 +1350,52 @@ class LibraryImportAttribute(string s) : System.Attribute { }
                 { Value: MethodDeclarationSyntax { Identifier.ValueText: "SomeFunction2" } }]));
     }
 
+    [Theory, CombinatorialData, WorkItem("https://github.com/dotnet/runtime/issues/105137")]
+    public void LocalFileAlias_AttributeUsedMultipleTimes2(bool withAttribute)
+    {
+        var source = $$"""
+            using System.Runtime.InteropServices;
+
+            namespace NetPlayground
+            {
+                using ImportAttribute = LibraryImportAttribute;
+
+                internal partial class TestImport
+                {
+                    [Import{{(withAttribute ? "Attribute" : "")}}("somedll.dll")]
```

Comment:
i wanted tests that go through both atltribute-lookup paths to make sure that we have no bugs in either.